src/org/usfirst/frc/team2335/robot/commands/ShootGroup.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShootGroup extends CommandGroup {
+    
+    public ShootGroup()
+    {
+    	DriverStation.reportWarning("Running", true);
```

Comment:
I still don't quite get how the text display stuff works and primarily where it shows up. Maybe document and also maybe show me.